src/condor_tests/ornithology-docs/source/guides/getting-started.rst

```diff
@@ -12,7 +12,7 @@ of tools for inspecting the state of an HTCondor pool.
 
 Ornithology is designed to be flexible and extensible.
 If you find yourself needing a new tool that might be useful in other
-contexts, feel free to add it the ``ornithology`` module itself.
+contexts, feel free to add the ``ornithology`` module itself.
```

Comment:
For the ornithology doc changes, I feel like we should be saying _feel free to add it to the ornithology module itself._ and not _feel free to add to the ornithology..._ . I read this portion of the docs as an invitation to expand the ornithology modules.